python/lsst/afw/math/functionLibrary.cc

```diff
@@ -61,8 +61,8 @@ void declarePolynomialFunctions(py::module &mod, const std::string & suffix) {
                BasePolynomialFunction2<ReturnT>>
                    clsPolynomialFunction2(mod, ("PolynomialFunction2" + suffix).c_str());
     /* PolynomialFunction2 Constructors */
-    clsPolynomialFunction2.def(py::init<unsigned int>());
     clsPolynomialFunction2.def(py::init<std::vector<double> const &>());
+    clsPolynomialFunction2.def(py::init<unsigned int>());
```

Comment:
I'm curious what happened here. It appears that two lines simply got swapped (here and above at line 50 and a definition at about line 81). If so I'd like to understand why the swap helped (so I can better understand C++).